.gitignore

```diff
@@ -4,4 +4,5 @@
 /node_modules
 
 *.log
-.DS_Store
\ No newline at end of file
+.DS_Store
+.history
```

Comment:
nit suggestion @AlemTuzlak , I also use the history plugin. To avoid having to add this in every single project I use, I've created a global gitignore and set it there: https://www.raulmelo.me/en/til/how-to-set-a-global-gitignore